x-pack/plugins/security_solution/public/cases/components/settings/connector.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import styled from 'styled-components';
+
+import { ConnectorTypeFields } from '../../../../../case/common/api/connectors';
+import { ConnectorCard } from './card';
+import { connectorsFieldLabels } from '.';
+
+interface ConnectorComponentProps {
+  name: string;
+  type: string;
+  fields: ConnectorTypeFields['fields'];
+}
+
+const MyFlexGroup = styled(EuiFlexGroup)`
+  margin-top: 0;
+  & *,
+  & img {
+    margin: 0;
+  }
+`;
+
+const ConnectorComponent: React.FC<ConnectorComponentProps> = ({ name, type, fields }) => {
+  const listItems = useMemo(
+    () =>
+      Object.entries(fields ?? {})
+        .filter(([key, value]) => value != null)
+        .map(([key, value]) => {
+          const connectorLabels = connectorsFieldLabels[type] ?? {};
+
+          return { title: connectorLabels[key] ?? '', description: value };
```

Comment:
value can be an array here, we need to check for that and if it's an array, list all the values.

Reply:
yes that's related to what I saw on the node side